src/power_grid_model/core/__init__.py

```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""
+Power Grid Model core library
+
+[deprecated]
+"""
```

Comment:
Shouldn't we expose here the (unintentionally previously exposed) core functionalities as well? Or we leave this blank because, as before, the user has to manually import the specific module from core that is wanted?

Reply:
this was already blank before, so there's no risk here. manual import, it is